docs/source/storage/general_storage.rst

```diff
@@ -1,34 +1,32 @@
 .. _general_storage:
 
+.. |ss| raw:: html
+
+    <strike>
+
+.. |se| raw:: html
+
+    </strike>
+
 *****************************
 General Data Storage Options
 *****************************
 The Research Computing (RC) team is responsible for the procurement and ongoing
 maintenance of several data storage options, including active and archive
 storage solutions. If you are affiliated with Northeastern, you can request
-one or more storage solutions to meet your storage needs. It is highly recommended
-that if you anticipate needing storage as part of a grant requirement,
-schedule a consultation with a Research Computing staff member to understand what
+one or more storage solutions to meet your storage needs. If you anticipate needing storage as part of a grant requirement,
+please schedule a consultation with an RC staff member to understand what
```

Comment:
Can we move the consultation link from the following sentence to: "schedule a consultation with an RC staff member". Then we can just delete the last sentence: "You can schedule an online consultation on the RC Website."

Reply:
I thought about trying to do it myself, but didn't want to mess anything up!

Reply:
No worries. I made the adjustments because I still had it open on my end.